pom.xml

```diff
@@ -519,6 +519,27 @@
         <version>${plugin.deploy.version}</version>
       </plugin>
 
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
```

Comment:
As long as git-commit-id-plugin binary is not included in Zeppelin binary package release, we don't need (and we shouldn't) add it in LICENSE_BIN. 

Same for source package. as long as it's not included in source package distribution, we don't need (and shouldn't) add it in LICENSE.